build.sbt

```diff
@@ -297,7 +287,23 @@ lazy val dockerSettings = {
       List(dockerVersion, updateLatest, latestBranch, dockerTagName)
         .map(tag => alias.withTag(tag.map(sanitize)))
         .distinct
-    },
+    }
+  )
+}
+
+lazy val distDockerSettings = {
+  val workingDir = "/opt/nussknacker"
```

Comment:
workingDir, defaultLinuxInstallLocation and exposedVolumes can be in common?

Reply:
Well, after Arek's comments I added a /processes volume specific to standalone. So now `exposedVolumes` is not the same for both images. Plus I need workingDir to construct entrypoint path, which is specific for each image. 

As for `defaultLinuxInstallLocation` it probably could, or even should be common. But it depends on workingDir, so I'd have to either make it a global constant in the file or define common settings as a function of workingDir instead of a simple Seq. Do you think it's worth it?